src/main/java/com/spaceuptech/space_api/sql/SQL.java

```diff
@@ -0,0 +1,71 @@
+package com.spaceuptech.space_api.sql;
+import com.spaceuptech.space_api.proto.Meta;
+import com.spaceuptech.space_api.utils.Config;
+import com.spaceuptech.space_api.utils.Transport;
+import com.spaceuptech.space_api.utils.Utils;
+
+public class SQL {
+    private Config config;
+    private String db;
+
+    public SQL(String db, Config config) {
+        this.db = db;
+        this.config = config;
+    }
+
+    // CREATE
+    public Insert insert(String collection) {
+        return new Insert(this.db, this.config, collection);
+    }
+
+
+    // READ
+    public Get get(String collection) {
+        return new Get(this.db, this.config, collection, "all");
```

Comment:
can we make `all`, `one`, `upsert`, etc. constants?

Reply:
Cool